Source/Controllers/MainViewControllers/TableStructure/SPTableStructure.m

```diff
@@ -2718,6 +2770,50 @@ + (SPFieldTypeHelp *)helpForFieldType:(NSString *)typeName
 	return nil;
 }
 
+- (IBAction)filterChanged:(NSSearchField *)sender
+{
+	if (sender == filterSearchField && [self filterFieldsWithString:sender.stringValue])
```

Comment:
Let's try to always use brackets {} for if statements from a style perspective
if () {
 ///thing
}

Reply:
Sounds good! Done.